tests/server/test_app.py

```diff
@@ -125,6 +160,22 @@ def test_invalid_thread_pinning():
     with pytest.raises(ValueError, match='Expected one of {"core","numa","none"}.'):
         _build_app(
             ServerConfig(
-                num_cores=1, num_workers=1, engine_thread_pinning="asdf", endpoints=[]
+                num_cores=1,
+                num_workers=1,
+                engine_thread_pinning="asdf",
+                endpoints=[],
+                loggers=None,
             )
         )
+
+
+@mock_engine(rng_seed=0)
+def test_default_logging_server_launches(engine_mock):
+    server_config = ServerConfig(
+        num_cores=1, num_workers=1, endpoints=[], loggers="default"
+    )
+    _build_app(server_config)
+
+    # check positive ping to expected prometheus server on port 6100
+    response = requests.get("http://127.0.0.1:6100")
```

Comment:
Are there other endpoints we are expecting to exist that we could throw in a quick test for?

Reply:
there are, working out what the ones we want to hit are and will update in the graphana PR

Reply:
Okay sounds good